developer/behat/getting-started.rst

```diff
@@ -91,6 +91,13 @@ Where ``<profile>`` is one of:
   above)
 - **sauce_labs**: Run the tests on Sauce Labs (see :doc:`sauce-labs`).
 
+.. note
+
+    If you want to run the tests on your local machine you have to make sure
+    that a Sulu instance is running on 127.0.0.1:8001. You can use the command
+    `env SYMFONY_ENV=dev ./app/console server:run 127.0.0.1:8001
```

Comment:
env SYMFONY_ENV is redundant. I just use it to switch from PROD and DEV. It defaults to DEV which is correct.